source/DSCResources/DSC_CMCollections/DSC_CMCollections.psm1

```diff
@@ -444,28 +608,46 @@ function Set-TargetResource
 
     .PARAMETER LimitingCollectionName
         Specifies the name of a collection to use as the default scope for this collection.
-        Limiting collection is not evaluated in Test and is only used
-        if the collection needs created.
 
     .PARAMETER Comment
         Specifies a comment for the collection.
 
-    .PARAMETER RefreshSchedule
-        Specifies a schedule that determines when Configuration Manager refreshes the collection.
+    .PARAMETER Start
+        Specifies the start date and start time for the maintenance window Month/Day/Year, example 1/1/2020 02:00.
+
+    .PARAMETER ScheduleType
+        Specifies the schedule type for the maintenance window.
+
+    .PARAMETER RecurInterval
+        Specifies how often the ScheduleType is run.
+
+    .PARAMETER MonthlyByWeek
+        Specifies week order for MonthlyByWeek schedule type.
+
+    .PARAMETER DayOfWeek
+        Specifies the day of week name for MonthlyByWeek and Weekly schedules.
+
+    .PARAMETER DayOfMonth
+        Specifies the day number for MonthlyByDay schedules.
+        Note specifying 0 sets the schedule to run the last day of the month.
 
     .PARAMETER RefreshType
         Specifies how Configuration Manager refreshes the collection.
         Valid values are: Manual, Periodic, Continuous, and Both.
 
-    .PARAMETER QueryRules
-        Specifies the name of the rule and the query expression that Configuration Manager uses to update collections.
-
     .PARAMETER ExcludeMembership
-        Specifies the collection name to exclude members from. If clients are in the excluded collection they will
+        Specifies the collection name to exclude clients from. If clients are in the excluded collection they will
```

Comment:
For the includemembership and excludemembership parameters, I would stop the description before the word clients. Ex: "Specifies the name of the collection to exclude."

Reply:
Will need to account for change in set, in the schema, and in the readme.